src/Configurator/DockerfileConfigurator.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex\Configurator;
+
+use Symfony\Flex\Recipe;
+
+/**
+ * Adds commands to a Dockerfile.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class DockerfileConfigurator extends AbstractConfigurator
+{
+    public function configure(Recipe $recipe, $config): void
+    {
+        $dockerfile = getcwd() . '/Dockerfile';
+        if (!file_exists($dockerfile) || $this->isFileMarked($recipe, $dockerfile)) {
+            return;
+        }
+
+        $this->write('Adding Dockerfile entries');
+
+        $lines = [];
+        foreach (file($dockerfile) as $line) {
+            $lines[] = $line;
+            if ("###> recipes ###\n" !== $line) {
```

Comment:
Sometimes this will be always true under Windows, because `$line` can have `\r\n` at the end

Reply:
No because this line is generated by Flex, and Flex always uses Unix EOLs.

Reply:
> No because this line is generated by Flex, and Flex always uses Unix EOLs.

`git` (or something else) might do EOL conversion... So that assumption is dangerous.

Reply:
Maybe use the same markers as Docker official images?

https://github.com/docker-library/php/blob/04e67dc6bb76783d2d1b64513b8f42961e578645/Dockerfile-debian.template#L115